src/exercise/dist/clients/item_client.js

```diff
@@ -40,4 +40,21 @@ class ItemClient {
         }
     }
 
+    async changeTaskStatus(taskId) {
+        try {
+            await axios.post(`${this.API_BASE}/status`, 
+                {
+                    "taskId": taskId
+                }, 
+                { 
+                    headers: {
+                        'Content-Type': 'application/json'
+                    } 
+                }
+            );
+        } catch(error) {
+            throw new Error();
+        }
+    }
+
```

Comment:
its better practice to send to what status you want to change, later on when you'l have multiple clients simultaneously you don't want the server to toggle, if the user checked it to done this is what you want to save